src/components/views/rooms/EventTile.js

```diff
@@ -50,17 +50,6 @@ var eventTileTypes = {
 
 var MAX_READ_AVATARS = 5;
 
-// Our component structure for EventTiles on the timeline is:
```

Comment:
The structure has changed now. I can create a new diagram, but it doesn't feel sensible to mockup presentation like this when the layout is a different concern within another repository. It's not so easy to change!